stdlib/src/builtin/string.mojo

```diff
@@ -381,19 +420,30 @@ fn _identify_base(str_ref: StringRef, start: Int) -> Tuple[Int, Int]:
 fn atol(str: String, base: Int = 10) raises -> Int:
     """Parses and returns the given string as an integer in the given base.
 
-    For example, `atol("19")` returns `19`. If base is 0 the the string is
-    parsed as an Integer literal, see: https://docs.python.org/3/reference/lexical_analysis.html#integers.
+    For example, `atol("32")` returns `32`, and `atol("FF", 16)` returns `255`. If base is set to 0, the string is parsed as an Integer literal, with the following considerations:
```

Comment:
I'm not sure how many characters these are here since I don't have my IDE ruler, but mojo goes for 80 characters. You can set it up in vscode or whatever IDE you use if you wish. But this and other docstring lines are definitely above 80